tests/lib/tools/tests.backup

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: tests.backup prepare"
+    echo "       tests.backup restore"
+}
+
+cmd_prepare() {
+    local BACKUP_PATH=${1:-$(pwd)}
```

Comment:
Slight preference for the default argument to be resolved in main section below.

Reply:
done